src/api/api.controller.ts

```diff
@@ -44,9 +51,15 @@ export class ApiController {
     @Body()
     loginUserData: LoginUserDto,
   ) {
-    return await this.userService.login({
+    const loginUser = await this.userService.login({
       email: loginUserData.email,
       password: loginUserData.password,
     });
+
+    return {
+      statusCode: 200,
```

Comment:
(suggestion; non-blocking): Nest offers an enum for these. I think it's the `HttpStatus` enum.